std_msgs/Header header
string ns
int32 id

Point2D[] points
